src/sidebar/components/action-button.js

```diff
@@ -0,0 +1,64 @@
+'use strict';
+
+const classnames = require('classnames');
+const propTypes = require('prop-types');
+const { createElement } = require('preact');
+
+const SvgIcon = require('./svg-icon');
+
+/**
+ * A button, with (required) text label and an (optional) icon
+ */
+function ActionButton({
+  icon = '',
+  isActive = false,
+  isPrimary = false,
+  label,
+  onClick = () => null,
+  useCompactStyle = false,
+}) {
+  return (
+    <button
+      className={classnames('action-button', {
+        'action-button--compact': useCompactStyle,
+        'action-button--primary': isPrimary,
+        'is-active': isActive,
+      })}
+      onClick={onClick}
+      aria-label={label}
```

Comment:
Since the button has a mandatory text label, is there any value in either `aria-label` or `title`?